plugins/amazonq/codetransform/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codemodernizer/utils/CodeTransformProjectUtilsTest.kt

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codemodernizer.utils
+
+import com.intellij.openapi.projectRoots.JavaSdkVersion
+import com.intellij.openapi.projectRoots.Sdk
+import com.intellij.openapi.roots.LanguageLevelProjectExtension
+import com.intellij.openapi.roots.ProjectRootManager
+import com.intellij.pom.java.LanguageLevel
+import com.intellij.testFramework.IdeaTestUtil.getMockJdk21
```

Comment:
resolve failure by making a new `tst-233+` folder and extracting test to that directory

Reply:
I don't understand the workaround. I need to make an entire new folder for all new tests? Just this one time?

Reply:
My local build is working

Reply:
The active IDE version in your local project is declared in `gradle.properties`
https://github.com/aws/aws-toolkit-jetbrains/blob/main/gradle.properties#L11

Occasionally we run into situations where implementation needs to be specific to a major version range. In this case your test references code that only exists in 233+, which can be merged back into the standard `src`/`tst` folders when we drop 232.

Bottom of https://github.com/aws/aws-toolkit-jetbrains/blob/main/REPOLAYOUT.md#subprojects has a brief description

Reply:
Thank you for the detailed documentation and explanation. I will let me team know about this as well during standup.

Updated. Cheers!